### ISSUE TITLE
Linux Chrome drag and drop transparency issue

### ISSUE THREAD

**Issue by TGCrystal** [OpenCircuits/OpenCircuits#592]
**Describe the bug**
When dragging and dropping a component onto the canvas from the itemnav, there is a white background that should be invisible. This occurs on Ubuntu on Chrome/Chromium based browsers.

**To Reproduce**
1. Go to OpenCircuits on Chrome on Linux
2. Drag a component from the itemnav to the canvas
3. The background of the circuit svg is white instead of transparent

**Screenshots**
![dragNDropBug](https://user-images.githubusercontent.com/29614966/109352415-d2876880-7848-11eb-811b-e1f29b4957d6.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.2 LTS
 - Browser: Chrome Version 88.0.4324.182 (Official Build) (64-bit), Brave Version 1.13.86 Chromium: 85.0.4183.102 (Official Build) (64-bit)

**Additional context**
This issue is present on the latest branch and on the current live site.

**Comment by TGCrystal** [comment]
I've done some more digging and found this stackoverflow question: https://stackoverflow.com/questions/33490122/transparent-background-for-html5-drag-and-drop-ghost-image
It linked to this chromium bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=550999
Which got merged into this bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=369209
Looking over that thread, it seems to be a problem with Nvidia's Linux drivers. The issue says it was closed, but I'm still having that transparency issue. I am using a Nvidia card, and when I disabled hardware acceleration in Chrome the issue went away. I will look to see if I can find a more recent report of someone having a similar issue.
